.github/workflows/publish-docs.yml

```
@@ -0,0 +1,60 @@
+
+      - name: 📦 Install dependencies
+        run: |
+          poetry install --with=docs
+
+      - name: ⚙️ Configure git for github-actions
+        run: |
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+
+      - name: 🚀 Deploy Development Docs
+        if: (github.event_name == 'push' && github.ref == 'refs/heads/develop') || github.event_name == 'workflow_dispatch')
```

actionlint (1.7.7)

parser did not reach end of input after parsing the expression. 1 remaining token(s) in the input: ")"

(expression)